themes/buildpacks/layouts/shortcodes/comparison-table.html

```diff
@@ -154,16 +193,26 @@
     <tr>
       <th class="p-3">Governance</th>
       <td class="border-left border-right">CNCF</td>
+      <td>Docker</td>
       <td>Red Hat</td>
       <td>Google</td>
       <td>Google</td>
     </tr>
+    <tr class="border-top bg-light">
+      <th class="p-3">Target image types</th>
```

Comment:
```suggestion
      <th class="p-3">Best for Building...</th>
```

Not sure this is a better title but at minimum we should use Title Case.